examples/tdengineTelemetryService/deployment/http-deviceshifu-deployment.yaml

```diff
@@ -33,4 +36,7 @@ spec:
         - name: deviceshifu-config
           configMap:
             name: http-configmap-0.0.1
+        - name: deviceshifu-secret
+          secret:
+            secretName: deviceshifu-secret
```

Comment:
How about set secret as `optional`? `Empty secret` will make the user more confused when using deviceshifu.